UPGRADE.md

```diff
@@ -0,0 +1,19 @@
+Upgrading Instructions for Yii Sentry
+=====================================
+
+Upgrade from 2.0
+-----------------------
+* Consult Sentry 4.x upgrade guide https://github.com/getsentry/sentry-php/blob/4.0.0/UPGRADE-4.0.md. Some notable changes are:
+  * Configuration options (`['yiisoft/yii-sentry']['options']`) changes: `send_attempts`, `ignore_errors`, `logger` and `enable_compression`.
+  * In case of custom di configuration of Http client, TransportFactoryInterface or logger you may need to update your di configuration like this:
```

Comment:
```suggestion
  * In case of custom DI configuration of HTTP client, `TransportFactoryInterface` or logger you may need to update your DI configuration like this:
```